package.json

```diff
@@ -110,7 +108,13 @@
     "standard": "8.6.0",
     "wd": "^1.1.3",
     "webpack-stream": "3.2.0",
-    "cross-env": "^3.1.4"
+    "cross-env": "^3.1.4",
+    "react": "15.4.2",
+    "react-dom": "15.4.2"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
```

Comment:
What would be the reason to support an older version of react 15? Seems totally fine to start asking for `^15.4.2`, since we included it out of the box for all this time already 😄